sandbox-tools/merge-repos/src/syncReposToStaging.ts

```diff
@@ -741,32 +870,90 @@ localGit.checkIsRepo().then(async (isRepo) => {
         await processRepos(reposToSyncAndMerge, async (repoName, repoDetails) => {
             log(`Merging ${repoName} from ${repoDetails.url} using ${repoDetails.mergeBranchName} into ${repoDetails.destFolder}`);
 
-            await addRemoteAndFetch(mergeGit, repoName, repoDetails);
-            branchCommitDetails[repoName] = await mergeRemoteIntoBranch(mergeGit, repoName, repoDetails);
-            //await removeRemote(mergeGit, repoName);
-        });
+            let sourceDetails = await getAndSyncSrcRepo(mergeGit, repoName,repoDetails);
 
-        // Now merge / move each repo into the staging location
-        console.log("Now merge repos into main merge staging")
-        await processRepos(reposToSyncAndMerge, async (repoName, repoDetails) => {
-            let commitDetails = await mergeBranchToMergeMaster(mergeGit, workingBranch, repoDetails, branchCommitDetails[repoName]);
+            log(`Adding Remote ${repoName} => ${sourceDetails.path} ${sourceDetails.branch}`)
+            // Add the new source repo as the remote
+            await addRemoteAndFetch(mergeGit, repoName, {
+                url: sourceDetails.path,
+                branch: sourceDetails.branch
+            });
+
+            branchCommitDetails[repoName] = {
+                committed: false,
+                message: "Merging " + sourceDetails.message
+            };
+
+            log("Merge Staged Repo into main merge staging");            
+            let mergeBranchName = repoName + "/" + sourceDetails.branch
+            let commitDetails = await mergeBranchToMergeMaster(mergeGit, mergeBranchName, workingBranch, repoDetails.destFolder, branchCommitDetails[repoName]);
             if (commitDetails.committed) {
                 prTitle += repoName + "; ";
                 if (prBody) {
                     prBody += "\n";
                 }
                 prBody += `# Changes from ${repoName}@${repoDetails.branch} (${repoDetails.url})\n${commitDetails.message}`;
-                createPr = true;
+                prRequired = true;
+            }
+
```

Comment:
Try and push the new prefixed tags to both the `<user>` fork and the master repo (the master will probably fail as I don't believe that the `opentelemetrybot` has the privileges to do so). But an admin running this locally will -- so we can cause the tags to end up in the main `sandbox` repo either via the GITHUB action or by running manually.